docs/requirements-doc.txt

```diff
@@ -11,7 +11,7 @@ sphinx
 sphinx-click
 sphinx_rtd_theme
 pandas
-modin[all]
+git+https://github.com/modin-project/modin.git
```

Comment:
Shouldn't we install Modin deps?

Reply:
added

Reply:
Since Modin has pandas and numpy as dependencies by default we probably should remove pandas and numpy from here to avoid possible dependency conflicts when installing.

Reply:
removed